### PR TITLE
rgw_sal_motr: [CORTX-31638] add log for motr object details

### PR DESCRIPTION
**Pull request by sjain09**
## Problem
There is no log that have s3 obj name with motr object details in single log line.

## Solution
Add log corelating s3 object key with motr obj details like oid, fid, lid.

Signed-off-by: saurabh jain <saurabh.jain2@seagate.com>

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by dharmendra-jyani — APPROVED** [review]

**Review by DPG17 — APPROVED** [review]

File: src/rgw/rgw_sal_motr.cc [hunk 1]
```diff
@@ -5156,8 +5162,8 @@ int MotrStore::create_motr_idx_by_name(string iname)
   if (rc != 0 && rc != -EEXIST) {
     ldout(cctx, 0) << __func__ << ": ERROR: index create failed, rc = " << rc << dendl;
     ADDB(RGW_ADDB_REQUEST_ID, addb_logger.get_id(), 
-	 RGW_ADDB_FUNC_CREATE_IDX_BY_NAME,
-	 RGW_ADDB_PHASE_ERROR);
+    RGW_ADDB_FUNC_CREATE_IDX_BY_NAME,
```
[the remark this answers is not in the document]
Comment (sjain09) [reply src/rgw/rgw_sal_motr.cc]: Updated.

File: src/rgw/rgw_sal_motr.cc [hunk 2]
```diff
@@ -3380,9 +3387,7 @@ int MotrAtomicWriter::complete(size_t accounted_size, const std::string& etag,
   }
   encode(attrs, bl);
   obj.meta.encode(bl);
-  ldpp_dout(dpp, 20) <<__func__<< ": lid=0x" << std::hex << obj.meta.layout_id
```
[the remark this answers is not in the document]
Comment (sjain09) [reply src/rgw/rgw_sal_motr.cc:3445]: This has been merged in the new log.

File: src/rgw/rgw_sal_motr.cc [hunk 3]
```diff
@@ -2309,9 +2309,9 @@ int MotrObject::open_mobj(const DoutPrefixProvider *dpp)
 
   if (meta.layout_id == 0) {
     ADDB(RGW_ADDB_REQUEST_ID, addb_logger.get_id(), 
-	 RGW_ADDB_FUNC_OPEN_MOBJ,
-	 RGW_ADDB_PHASE_DONE);
-
+    RGW_ADDB_FUNC_OPEN_MOBJ,
```
[the remark this answers is not in the document]
Comment (sjain09) [reply src/rgw/rgw_sal_motr.cc]: Updated.

File: src/rgw/rgw_sal_motr.cc [hunk 4]
```diff
@@ -3791,9 +3796,10 @@ int MotrMultipartUpload::list_parts(const DoutPrefixProvider *dpp, CephContext *
 
     ldpp_dout(dpp, 20) << __func__ << ": part_num=" << info.num
                                              << " part_size=" << info.size << dendl;
-    ldpp_dout(dpp, 20) << __func__ << ": meta:oid=[" << meta.oid.u_hi << "," << meta.oid.u_lo
-                                              << "], meta:pvid=[" << meta.pver.f_container << "," << meta.pver.f_key
-                                              << "], meta:layout id=" << meta.layout_id << dendl;
+    ldpp_dout(dpp, 20) << __func__ << ": key=" << mp_obj.get_key() << ", meta:oid=[0x" << std::hex << meta.oid.u_hi
```
[the remark this answers is not in the document]
Comment (sjain09) [reply src/rgw/rgw_sal_motr.cc:4080]: Updated.

**Pull request merged**